src/graphql/thegraph/apollo.ts

```diff
@@ -3,7 +3,7 @@ import { ChainId } from '@uniswap/sdk-core'
 
 import store from '../../state/index'
 
-const CHAIN_SUBGRAPH_URL: Record<number, string> = {
+export const CHAIN_SUBGRAPH_URL: Record<number, string> = {
```

Comment:
maybe this is the right file for apollo client instantiation?

Reply:
see my comment on needing dynamic client based on chain to search for https://github.com/Uniswap/interface/pull/7250#discussion_r1318936410